DEFAULT_CONFIG.json5

```diff
@@ -27,6 +27,12 @@
       ////
       // localhost_only: true,
 
+      ////
+      //// shm_enabled: If set to true, the DDS implementation will use Iceoryx shared memory.
```

Comment:
Specify that the bridge has to be built with `dds_shm` feature for this configuration to be applied

Reply:
Requested information added. I've also added the same warning to the Configuration section in the readme.